docs/tools/aiven-console/howto/create-manage-teams.rst

```diff
@@ -54,3 +56,24 @@ To add projects and roles to a team:
 #. Click **Add project to team**.
 
 You can edit the permissions or delete the project from this team by clicking the more options menu for the project.
+
+.. _migrate_teams_to_groups:
+
+Migrate teams to groups
+------------------------
+
+To get started using organization groups, replace your existing teams with groups:
+
+#. In the organization, click **Admin** 
+
+#. Click **Organization** and on the **Teams** tab view each team to make a note of: 
+
+* which users are members of the team
+* which projects the team is assigned to
```

Comment:
This and the point below are more naturally done after creating the group. Therefore I suggest to move this after "Enter the name of one of the teams and assign the same users to this group. Do this for each of your organization's teams." and add an extra step which says navigate the the project.

Reply:
I think this process is a bit more efficient, though?  It's easier to get all of the information about a team from Admin > Org > Teams  instead of navigating to each project individually. Also, when they create the group they can add the team members right away instead of doing it in another step.

Reply:
It also slightly duplicates the instructions on the linked page if we tell them to go to the project.